core/storage/topic/gae_models_test.py

```diff
@@ -155,6 +155,11 @@ def test_get_deletion_policy(self):
             topic_models.TopicSummaryModel.get_deletion_policy(),
             base_models.DELETION_POLICY.KEEP_IF_PUBLIC)
 
+    def test_has_reference_to_user_id(self):
+        self.assertFalse(
+            topic_models.TopicSummaryModel.has_reference_to_user_id('any_id'))
+
```

Comment:
Drop one of these newlines (there should only be two newlines above top-level class definitions).